support/package-builder/prepare-build-root.sh

```diff
@@ -47,15 +47,17 @@ if mountpoint ${BUILDROOT}/dev	>/dev/null 2>&1; then umount ${BUILDROOT}/dev; fi
 #Untar tools build to buildroot
 #
 
-echo "Uncompressing the tools tar"
-
-tar -C ${BUILDROOT} -xvf $TOOLS_PATH/tools-build.tar >/dev/null
-# YUSTAS_FIXME: rpm still depends on /tools/lib/... Remove this dependency.
-ln -sf ${BUILDROOT}/tools /tools
+if [ "x$TOOLS_ARCHIVE" = "xminimal" ] ; then
+    echo "Uncompressing the tools tar (minimal)"
+    tar -C ${BUILDROOT} -xzf $TOOLS_PATH/tools.tar.gz >/dev/null
```

Comment:
And I think you might intended to untar in /tools folder instead of ${BUILDROOT} so that you could reuse it over and over again below.